libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/preview/PreviewImageFragment.kt

```diff
@@ -271,14 +271,16 @@ class PreviewImageFragment : Fragment(R.layout.preview_image_fragment) {
         // TODO: Temporarily added if check to fix this occasional crash
         // https://stackoverflow.com/q/51060762/193545
         if (navController.currentDestination?.id == R.id.preview_dest) {
-            navController.navigate(
-                    PreviewImageFragmentDirections.actionPreviewFragmentToCropFragment(
-                            inputFilePath,
-                            outputFileExtension,
-                            shouldReturnToPreviewScreen
-                    ),
-                    navOptions
-            )
+            outputFileExtension?.let {
```

Comment:
Great find, @ParaskP7. I tested the scenario and indeed the issue is resolved by the solution you mentioned. Thanks for linking the Android [issue](https://issuetracker.google.com/issues/198689811). 
Also thanks @oguzkocer for updating the library. 🙇‍♀️ 

We should  
- Remove `null`-check added for `outputFileExtension` in f73798c.
- Add `android:defaultValue="@null"` to `outputFileExtension` argument within the navigation graph for both the fragments: `preview_dest` and `crop_dest`.

@oguzkocer, let me know if it can be handled as part of this PR itself or if I should create another PR with a fix. 🙏

Reply:
Thank for the confirmation and testing this solution @ashiagr ! 🙇 🚀

Reply:
Thank for the confirmation and testing this solution @ashiagr ! 🙇 🚀

Reply:
Changes in 51af542 look good to me, thanks @oguzkocer!

Reply:
Thank you for the changes @oguzkocer ! 🚀 🌟